cmd/kubeapps-apis/server/server.go

```diff
@@ -65,13 +66,48 @@ func CreateRequestLogger() func(ctx context.Context, req interface{}, info *grpc
 	}
 }
 
+func getLogLevelOfEndpoint(endpoint string) klogv2.Level {
+
+	// Add all endpoint function names which you want to suppress in interceptor logging
+	supressLoggingOfEndpoints := []string{"GetConfiguredPlugins"}
+	var level klogv2.Level
+
+	// level=3 is default logging level
```

Comment:
The default log level is 3 if the user hasn't set a `-v` option on the cli... don't we need to default to whatever the user has specified (which will be 3 if nothing was specified)?

Reply:
I don't think we should complicate more by using the user settings (--v).

Reply:
Sorry, I confused this in my head with something it is not... I was thinking we were deciding here what the default log level would be for the app, but it's not, it's just the level that will be used for the interceptor. Carry on :)